src/test/unit/wrappers/http/test_http_parser.py

```diff
@@ -122,18 +127,191 @@ def test_lambda_parser_resource_name(uri, resource_name):
+        b"QueueUrl=https%3A%2F%2Fsqs.us-west-2.amazonaws.com%2F449953265267%2Fsagivdeleteme-node-sqs"
+        b"&MessageBody=%7B%0A%20%20%20%20%22body%22%3A%20%22test1%22%0A%7D"
+        b"&MessageAttribute.1.Name=AttributeName"
+        b"&MessageAttribute.1.Value.StringValue=Attribute%20Value"
+        b"&MessageAttribute.1.Value.DataType=String"
+        b"&Action=SendMessage"
+        b"&Version=2012-11-05",
+        # Send batch message request to SQS (on record in the batch)
+        b"QueueUrl=https%3A%2F%2Fsqs.us-west-2.amazonaws.com%2F449953265267%2Fsagivdeleteme-node-sqs"
+        b"&SendMessageBatchRequestEntry.1.Id=1"
+        b"&SendMessageBatchRequestEntry.1.MessageBody=Message%201"
+        b"&SendMessageBatchRequestEntry.2.Id=2"
+        b"&SendMessageBatchRequestEntry.2.MessageBody=Message%202"
+        b"&Action=SendMessageBatch"
+        b"&Version=2012-11-05",
+    ],
+)
+def test_sqs_xml_parse_resource_name(request_body):
+    queue_url = "https://sqs.us-west-2.amazonaws.com/449953265267/sagivdeleteme-node-sqs"
+    http_request = HttpRequest(host="dummy", method="POST", uri="", headers={}, body=request_body)
+    parsed_request = SqsXmlParser().parse_request(http_request)
+    assert parsed_request["info"]["resourceName"] == queue_url
+
+
 @pytest.mark.parametrize(
     "body",
     [
         b'<?xml version="1.0"?><SendMessageBatchResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/"><SendMessageBatchResult><SendMessageBatchResultEntry><Id>123</Id><MessageId>85dc3997-b060-47bc-9d89-c754d7260dbd</MessageId><MD5OfMessageBody>485b9ada0d1f06d60d71145304704c27</MD5OfMessageBody></SendMessageBatchResultEntry></SendMessageBatchResult><ResponseMetadata><RequestId>41295a06-b432-55b5-a8aa-00e764c8b9cf</RequestId></ResponseMetadata></SendMessageBatchResponse>',
         b'<?xml version="1.0"?><SendMessageResponse xmlns="http://queue.amazonAwsParser.com/doc/2012-11-05/"><SendMessageResult><MessageId>85dc3997-b060-47bc-9d89-c754d7260dbd</MessageId><MD5OfMessageBody>c5cb6abef11b88049177473a73ed662f</MD5OfMessageBody></SendMessageResult><ResponseMetadata><RequestId>b6b5a045-23c6-5e3a-a54f-f7dd99f7b379</RequestId></ResponseMetadata></SendMessageResponse>',
     ],
 )
-def test_sqs_parser_message_id(body):
-    response = SqsParser().parse_response("dummy", 200, {}, body=body)
+def test_sqs_xml_parser_message_id(body):
+    response = SqsXmlParser().parse_response("dummy", 200, {}, body=body)
     assert response["info"]["messageId"] == "85dc3997-b060-47bc-9d89-c754d7260dbd"
 
 
+@pytest.mark.parametrize(
+    "response_body, message_id",
+    [
+        (
+            # Send single message to SQS response
+            b"{"
+            b'   "MD5OfMessageAttributes":"6e6aba56e93b3ddfdfe3fa28895feece",'  # pragma: allowlist secret
+            b'   "MD5OfMessageBody":"0d40eb1479f7e61b1a1c7a425c3949e4",'  # pragma: allowlist secret
+            b'   "MessageId":"c5aca29a-ff2f-4db5-94c3-90523d1ed4ca"'  # pragma: allowlist secret
```

Comment:
usually I prefer to avoid random strings for readability. You can change it to any arbitrary value 🤷 
Then, it will be easier to understand that line 180 equals exactly line 178.
I would suggest using "aaaaa" and "bbbb"

Reply:
Done